polled_camera/src/publication_server.cpp

```diff
@@ -148,7 +151,7 @@ std::string PublicationServer::getService() const
 
 PublicationServer::operator void*() const
 {
-  return (impl_ && impl_->isValid()) ? (void*)1 : (void*)0;
+  return (impl_ && impl_->isValid()) ? reinterpret_cast<void*>(1) : reinterpret_cast<void*>(0);
```

Comment:
Orthogonal to resolving the linter complaint, does anyone have an idea as to why we would want to return a bogus pointer here in case the implementation is valid?

Reply:
That looks like a recipe for segfaults, to me. It must be being treated as an error code of some kind, I guess?

Reply:
Since this package hasn't been ported to ROS 2, I've dropped changes to it in this PR.